src/manager/agent.manager.service.ts

```diff
@@ -186,9 +184,6 @@ export class AgentManagerService {
                 const connectionData = await this.createConnection(agentId, agentData.adminPort, adminApiKey);
```

Comment:
I think we can remove the this.cache.get() above sinnce the admin key is passed inn